src/GT.UI/Views/Listing/ListingOverview.cshtml

```diff
@@ -1 +1,65 @@
+@inject SignInManager<ApplicationUser> SignInManager
+@inject UserManager<ApplicationUser> UserManager
 
+@model List<GT.Core.DTO.Impl.ListingOverviewDTO>
+@{
+	ViewData["Title"] = "Annonser";
+}
+<h1>@ViewData["Title"]</h1>
+
+@*public string? Id { get; set; }
+	public string? ListingTitle { get; set; }
+	public string? EmployerName { get; set; }
+	public int? SalaryMin { get; set; }
+	public int? SalaryMax { get; set; }
+	public string? JobTitle { get; set; }
+	public string? Location { get; set; }
+	public bool? FTE { get; set; }
+	public DateTime? CreatedDate { get; set; }
+	public string? ExperienceLevel { get; set; }*@
```

Comment:
Should these commented props be removed?

Reply:
Comment added.